src/routes/CardAdd/index.tsx

```diff
@@ -30,15 +33,30 @@ const CardAdd = () => {
   const cardSecurityCode = useCardNumberData(SECURITY_CODE_LENGTH_MAX)
   const cardPassword = useCardNumberData(PASSWORD_LENGTH_MAX)
 
-  const validationNavigate = () => {
+
+  const isValidationCardData = () => {
```

Comment:
아래처럼 변경 되면 변수명을 읽는게 더 자연스러워질거 같네요

```suggestion
  const isValidCardData = () => {
```

Reply:
변수명 변경했습니다!